spec/01-lexical-syntax.md

```diff
@@ -71,7 +71,7 @@ The rules for pattern matching further distinguish between
 _variable identifiers_, which start with a lower case letter
 or `_`, and _constant identifiers_, which do not.
 
-For this purpose, lower case letters doesn't only include a-z,
+For this purpose, lower case letters include not only include a-z,
```

Comment:
There are two `include` words in here...

Reply:
Removed extra include.